src/Context/playlistContext.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState, useEffect, createContext } from "react";
+import { useFetch } from "../CustomHooks";
+
+export const PlaylistContext = createContext([]);
+
+const token = localStorage.getItem("user")
```

Comment:
May be instead of the ternary operator you can directly use JSON.parse(localStorage.getItem("user"))?.token